BusBooking.cls

```diff
@@ -0,0 +1,140 @@
+public with sharing class BusBooking {
+
+public List<SelectOption> stops{get;set;}
+public String selectedstops{get;set;}
+public Account acc{get;set;}
+public  List<Route_Stop__c> lstroutestop{get; set;}
+public Integer nooftickets{get;set;}
+public List<SelectOption> capacitylist{get;set;}
+Vehicle_Route__c vehroute{get;set;}
+public String recid{get;set;}
+public Double doubcapacity{get;set;}
+public boolean availablesection{get;set;}
+public boolean notavailablesection{get;set;}
+public Double doub{get; set;}
+
+ 
+   public BusBooking(ApexPages.StandardController controller) { 
+     acc =(Account)controller.getRecord();
+     lstroutestop= new List<Route_Stop__c>();  
+     getTicketslist();
+     availablesection =false;
+     }  
+    
+   
+ /** List to display the number of tickets**/
+   public List<SelectOption> getTicketslist() {
+    capacitylist = new List<SelectOption>();
+    capacitylist.add(new SelectOption('0','0'));
+    capacitylist.add(new SelectOption('1','1'));
+    capacitylist.add(new SelectOption('2','2'));
+    capacitylist.add(new SelectOption('3','3'));
+    capacitylist.add(new SelectOption('4','4'));
+    capacitylist.add(new SelectOption('5','5'));
+    capacitylist.add(new SelectOption('6','6'));
+    capacitylist.add(new SelectOption('7','7'));
+    capacitylist.add(new SelectOption('8','8'));
+    capacitylist.add(new SelectOption('9','9'));
+    capacitylist.add(new SelectOption('10','10'));
+    return capacitylist;
+   }
+   
+   
+   /** List to display the schoolstops**/
+    public List<SelectOption> getStopList() {
+     stops= new List<SelectOption>();
+     stops.add(new SelectOption('None','None'));
+     stops.add(new SelectOption('Symbiosis School', 'Symbiosis School')); 
+     stops.add(new SelectOption('DAV School', 'DAV School'));
+     stops.add(new SelectOption('Army Public School', 'Army Public School')) ;  
+     return stops;  
+      } 
+      
+     
+    /** Get routes method to display the routes based on google api**/    
+     public void getroutes(){
```

Comment:
Divide this single method into multiple methods
- move callout code to utility class method
- move json parser code to utlilty class method